srepanel/web/request_access.go

```diff
@@ -0,0 +1,87 @@
+package web
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"go.mkw.re/ghidra-panel/common"
+	"go.mkw.re/ghidra-panel/discord"
+)
+
+func (s *Server) handleRequestAccess(wr http.ResponseWriter, req *http.Request) {
+	if req.Method != http.MethodPost {
+		http.Error(wr, "Method not allowed", http.StatusMethodNotAllowed)
+		return
+	}
+
+	ident, ok := s.checkAuth(req)
+	if !ok {
+		http.Error(wr, "Not authorized", http.StatusUnauthorized)
+		return
+	}
+
+	if err := req.ParseForm(); err != nil {
+		http.Error(wr, "Bad request", http.StatusBadRequest)
+		return
+	}
+
+	// TODO query ACL to ensure user account exists
```

Comment:
I don't understand this TODO comment ... If the username doesn't exist in the UserManager, we can just add them before authorizing.